tests/integration/test_system_app_creator.py

```diff
@@ -0,0 +1,37 @@
+from .common import random_str
+import time
+
+
+def test_system_app_creator(admin_mc, admin_system_pc):
+    client = admin_mc.client
+    provider_name = random_str()
+    access = random_str()
+    secret = random_str()
+    globaldns_provider = \
+        client.create_global_dns_provider(
+                                         name=provider_name,
+                                         rootDomain="example.com",
+                                         route53ProviderConfig={
+                                             'accessKey': access,
+                                             'secretKey': secret})
```

Comment:
We can use remove_resource so this provider resource gets deleted after test runs